sc/SCVim.sc

```diff
@@ -23,13 +23,30 @@
 SCVim.generateTagsFile();
 */
 
+Document {
```

Comment:
@grirgz `Document` class should use TABS instead of spaces for indention

Reply:
done